src/files-footer-detail.tsx

```diff
@@ -122,3 +128,41 @@ export const FilesFooterDetail = ({
         </div>
     );
 };
+
+const UserGroupPopover = ({ file }: { file: FileInfo }) => (
+    <Popover
+        id="files-footer-usergroup-popover"
+        hasAutoWidth
+        bodyContent={
+            <DescriptionList
+                isHorizontal
+                isCompact
+            >
+                <DescriptionListGroup key="user">
+                    <DescriptionListTerm>
+                        User
+                    </DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {file.user}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup key="group">
+                    <DescriptionListTerm>
+                        Group
+                    </DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {file.group}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+            </DescriptionList>
+        }
+    >
+        <Button
+            variant="link"
+            isInline
+            component="pre"
+        >
+            {file.user}:{file.group}
```

Comment:
This needs to be a little more complicated and simplify showing only the user when the user and group are equal. We do this already in `FileOwnership` in the files-card-body.tsx

Reply:
I agree with this. (Thanks for mentioning it!)